receiver/prometheusremotewritereceiver/receiver_test.go

```diff
@@ -267,6 +267,80 @@ func TestTranslateV2(t *testing.T) {
 			}(),
 			expectedStats: remote.WriteResponseStats{},
 		},
+		{
+			name: "TODO",
+			request: &writev2.Request{
+				Symbols: []string{
+					"",
+					"__name__", "teste", // 1, 2
+					"job", "service-x/test", // 3, 4
+					"instance", "107cn001", // 5, 6
+					"otel_scope_name", "scope1", // 7, 8
+					"otel_scope_version", "v1", // 9, 10
+					"d", "e", // 11, 12
+					"foo", "bar", // 13, 14
+					"f", "g", // 15, 16
+					"seconds", "help text about seconds", // 17, 18
+					"milliseconds", "help text about milliseconds", // 19, 20
+				},
+				Timeseries: []writev2.TimeSeries{
+					{
+						Metadata:   writev2.Metadata{Type: writev2.Metadata_METRIC_TYPE_GAUGE, UnitRef: 17, HelpRef: 18},
+						LabelsRefs: []uint32{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12},
+						Samples:    []writev2.Sample{{Value: 1, Timestamp: 1}},
+					},
+					{
+						Metadata:   writev2.Metadata{Type: writev2.Metadata_METRIC_TYPE_GAUGE, UnitRef: 17, HelpRef: 18},
+						LabelsRefs: []uint32{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12},
+						Samples:    []writev2.Sample{{Value: 2, Timestamp: 2}},
+					},
+					{
+						Metadata:   writev2.Metadata{Type: writev2.Metadata_METRIC_TYPE_GAUGE, UnitRef: 19, HelpRef: 20},
+						LabelsRefs: []uint32{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 13, 14},
+						Samples:    []writev2.Sample{{Value: 3, Timestamp: 3}},
+					},
+				},
+			},
+			expectedMetrics: func() pmetric.Metrics {
+				expected := pmetric.NewMetrics()
+				rm1 := expected.ResourceMetrics().AppendEmpty()
+				rmAttributes1 := rm1.Resource().Attributes()
+				rmAttributes1.PutStr("service.namespace", "service-x")
+				rmAttributes1.PutStr("service.name", "teste")
+
+				sm1 := rm1.ScopeMetrics().AppendEmpty()
+				sm1.Scope().SetName("scope1")
+				sm1.Scope().SetVersion("v1")
+
+				metrics1 := sm1.Metrics().AppendEmpty()
+				// Not sure where the metrics name are defined or coming from.
+				// I think that the reference to this metrics metadata can be found here: https://opentelemetry.io/docs/specs/otel/compatibility/prometheus_and_openmetrics/#metric-metadata
+				// Or maybe here: https://opentelemetry.io/docs/specs/otel/compatibility/prometheus_and_openmetrics/#metric-metadata
+				metrics1.SetName("TODO")
+				metrics1.SetUnit("seconds")
+				metrics1.SetDescription("help text about seconds")
```

Comment:
Adding this comment, because I noticed in docs that we should assign these values. Probably we will  remove it after discuss where is the best place to put the rules.

Reply:
This one that you mentioned is not related to ResourceMetrics -> Attributes -> service.name?

Sorry if I express myself wrong. But here, I'm talking about the attributes related to metrics1. They could be Name, Unit or Description. Taking a look into the code, we aren't assigning these values anywhere and if I understand right, they should be equal to the values that we define on Metadata -> `UnitRef` and `HelpRef`. But I didn't find the value that should be passed in the SetName...

What you are saying is that the value inside the metrics1.SetName should be equal to the one defined in `__name__`?